src/components/Categories/MainCategoryTile.vue

```diff
@@ -1,59 +1,56 @@
 <template>
 	<div v-if="tileSize === 'large'" class="tw-mb-6">
 		<img class="tw-rounded tw-w-full"
-			:src="image"
+			:src="categoryImage"
 		>
 		<div class="tw-text-center tw-flex tw-justify-between">
-			<h3 class="tw-mt-3 tw-mb-3">
-				Lend to women
+			<h3 class="tw-mt-2 tw-mb-2">
+				Lend to {{ categoryName }}
 			</h3>
-			<h4 class="tw-mt-3 tw-mb-3">
-				258 loans
+			<h4 class="tw-mt-2 tw-mb-2">
+				{{ numberLoans }} loans
 			</h4>
 		</div>
 		<div>
-			<span>
-				Help women around the world pursue their aspirations and gain
-				decision-making power in their homes and communities.
+			<span class="truncate">
+				{{ categoryDescription }}
 			</span>
 		</div>
 	</div>
 	<div v-else-if="tileSize === 'medium'" class="tw-mb-6">
```

Comment:
You can reduce some duplicated code here since the medium tile and large tile have the same layout